src/lib/encoder/test/EncoderRoundTripTest.cpp

```diff
@@ -67,7 +67,7 @@ TEST_CASE( "EncoderRoundTripTest/testFountain.SinkMismatch", "[unit]" )
 	enc.set_legacy_mode();
 	assertEquals( 1, enc.encode_fountain(inputFile, outPrefix) );
 
-	uint64_t hash = 0xaecc8c00efce8c28;
+	uint64_t hash = 0xaecc8808efce9c08;
```

Comment:
since the padding changes, so does the output of these small file encodes :no_mouth: